src/components/TaskList.js

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import { useState, useEffect } from "react";
+import { fetchData, sortDateTime } from "./../utils/utils";
+import AddTaskButton from "./../components/AddTaskButton";
+import {
+	MainCont,
+	TaskBox,
+	DateText,
+	Description,
+	TaskTitle,
+	TaskCont,
+	CheckBox,
+	Icon,
+} from "../styled-components/reusables";
+
+import tick from "./../images/tick.svg";
+
+export default function TaskList({ url }) {
+	const [tasks, setTasks] = useState(null);
+
+	useEffect(() => {
+		resolveTaskData();
+	}, []);
+
+	function resolveTaskData() {
+		fetchData(`${url}`)
```

Comment:
I haven't noticed it while pairing, but I think there is a repetition going in the `fetchData`. `fetchData` itself resolves the promise and catches errors, I think that the `then` and `catch` below aren't necessary. Have you tried taking them out?

Reply:
The tests fail when I remove the then and catch statements